trunner/harness/plo.py

```diff
@@ -124,14 +128,39 @@ def erase(self, device: str, offset: int, size: int, timeout: Optional[int] = No
             self.send_cmd("YES!")
             self.dut.expect_exact("Erased", timeout=timeout)
         except pexpect.TIMEOUT as e:
-            raise PloError("Wrong erase command output!", cmd=cmd, output=self.dut.before) from e
+            raise PloError(
+                "Wrong erase command output!", cmd=cmd, output=self.dut.before
+            ) from e
 
         try:
             self._assert_prompt()
         except PloError as e:
             e.cmd = cmd
             raise e
 
+    def cleanmarkers(self, device: str, clean: bool, cleanmarkers_args: tuple, timeout):
+        """Performs set jffs2 cleanmarkers command."""
+
+        if clean is True:
+            x = "-e"
+        else:
+            x = ""
```

Comment:
Please do it like in other commands, so `e = "-e" if erase else ""`. Above that, why did you use x?

Reply:
Not applied.